ResultTests/ResultTests.swift

```diff
@@ -31,10 +31,21 @@ final class ResultTests: XCTestCase {
 	}
 
 	func testErrorsIncludeTheCallingFunction() {
-		let function = __FUNCTION__
-		XCTAssertEqual(Result<(), NSError>.error().function ?? "", function)
+//		let function = __FUNCTION__
+//		XCTAssertEqual(Result<(), NSError>.error().function ?? "", function)
```

Comment:
Do these tests segfault because of the usage of `Result` or because of the `XCTAssertEqual`? I’d like to be cautious that we’re not just moving a segfault from the API to API consumers.